api/api.go

```diff
@@ -120,12 +125,36 @@ func (h *httpHandler) initRouter() {
 	}
 }
 
-func makeHandler(ctrl libnetwork.NetworkController, fct processor) http.HandlerFunc {
+const (
+	// only one Accept value for now
+	acceptRegex  = `^application/vnd\.libnetwork\.v\d\.\d\+json$`
```

Comment:
Yeah, let's wait for some more comments on the proper name.
 
Regarding the registration with IANA, at first glance I feel it is not an issue in our context, client - server in a local environment, but I will check.

Github follows the same approach:
https://developer.github.com/v3/media/
and I was not able to find its type in the IANA list
http://www.iana.org/assignments/media-types/media-types.xhtml

I will do some more research on this.
Thanks.